main.py

```diff
@@ -270,12 +271,22 @@ def workers_edit(worker_id):
                 Worker.phone: request.form["phone"] or None,
                 Worker.notes: request.form["notes"],
                 Worker.active: bool(request.form.get("active")),
+                Worker.organizing_dept_id: Department.get(request.form["organizing_dept"] == Department.name).id,
```

Comment:
So this is a bit awkward. But we are storing department ids rather than the names so we need a way to translate between them.